src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -221,6 +222,8 @@ class InferenceProfiler {
   /// should be collected.
   /// \param overhead_pct_threshold User set threshold above which the PA
   /// overhead is too significant to provide usable results.
+  /// \param collector Collects all of the raw data from experiments into a
```

Comment:
"Collects all" -- this is an action/verb. I would not expect that for an arg description.

Reply:
Reworded 👍